README.md

```diff
@@ -1,4 +1,5 @@
 # bookend
+[![video_ss](figures/video_screenshot.png)](https://www.youtube.com/watch?v=P1Sq7T9PvP0)
```

Comment:
Is it possible to decrease the size of the figure in line? It's kind of big. This suggestion may work, I'm not sure.

```suggestion
[![video_ss](figures/video_screenshot.png =250x)](https://www.youtube.com/watch?v=P1Sq7T9PvP0)
```

Reply:
Thanks. I used html tags to scale it intead of github MD.

Reply:
Looks good